docs/readmes/contributing/contribute_conventions.md

```diff
@@ -194,6 +194,53 @@ Orc8r's cloud code has some basic [CI lint checks](https://github.com/magma/magm
 - When import aliasing is required, prefer to alias with `snake_case` rather than `camelCase`
 - Prefer readable code over rigid adherence to max line lengths. Capping around 140 characters feels about right.
 
+### Python
+
+In general, we follow the [PEP 8 style guide](https://www.python.org/dev/peps/pep-0008/). 
+
+**Type Annotation**
+
+- All new code should be fully type-annotated. 
+  - For reference, please look at this [type hints cheat sheet for Python 3](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html). 
+
+**Documentation**
+
+- Document all public functions and *keep those docs up to date* when you make changes. 
+- We use [Google style docstrings](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods) in our codebase. 
+  - For VSCode users, [Python Docstring Generator](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) plugin is recommended. 
+
+Example:
+```
+def foo(arg1: str) -> int:
+    """Returns the length of arg1.
+
+    Args:
+        arg1 (str): string to calculate the length of
+
+    Returns: the length of the provided parameter
+    """
+    return len(arg1)
+```
+
+**Linter**
+
+- For mandatory lint checks, we have a unit test that runs [Pylint](https://pypi.org/project/pylint/) on all gateway services. 
+  - On CI, the check gets run as part of the `lte-test` job
+- Additionally, we have a [reviewdog](https://github.com/reviewdog/reviewdog) based lint checker with [wemake-python-styleguide](https://wemake-python-stylegui.de/en/latest/) enabled to aid the code review process. 
```

Comment:
"- Reviewdog linter using wemake-python-styleguide that comments on open pull requests"